plugins/modules/route53.py

```diff
@@ -360,136 +347,78 @@
       - 0 issue "ca.example.net"
       - 0 issuewild ";"
       - 0 iodef "mailto:security@example.com"
-
 '''
 
-import time
-import distutils.version
 
 try:
-    import boto
-    import boto.ec2
-    from boto.route53 import Route53Connection
-    from boto.route53.record import Record, ResourceRecordSets
-    from boto.route53.status import Status
+    import botocore
 except ImportError:
-    pass  # Handled by HAS_BOTO
+    pass  # Handled by AnsibleAWSModule
+
+from ansible.module_utils._text import to_native
 
 from ansible_collections.amazon.aws.plugins.module_utils.core import AnsibleAWSModule
-from ansible_collections.amazon.aws.plugins.module_utils.ec2 import get_aws_connection_info
-from ansible_collections.amazon.aws.plugins.module_utils.ec2 import HAS_BOTO
+from ansible_collections.amazon.aws.plugins.module_utils.core import is_boto3_error_message
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import AWSRetry
+
+from operator import itemgetter
+
+MAX_AWS_RETRIES = 10  # How many retries to perform when an API call is failing
+WAIT_RETRY = 5  # how many seconds to wait between propagation status polls
+
+
+def remove_none_values_from_dict(dictionary):
+    return dict(filter(lambda i: i[1] is not None, dictionary.items()))
```

Comment:
We now have `ansible_collections.amazon.aws.plugins.module_utils.core.scrub_none_parameters()` which does the same thing